core/src/components/textarea/textarea.tsx

```diff
@@ -530,17 +536,37 @@ Developers can use the "legacy" property to continue using the legacy form marku
 
   private renderLabel() {
     const { label } = this;
-    if (label === undefined) {
-      return;
-    }
 
     return (
-      <div class="label-text-wrapper">
-        <div class="label-text">{this.label}</div>
+      <div
+        class={{
+          'label-text-wrapper': true,
+          'label-text-wrapper-hidden': !this.hasLabel,
+        }}
+      >
+        {label === undefined ? <slot name="label"></slot> : <div class="label-text">{label}</div>}
```

Comment:
The screen reader (Voice Over) is reading the label three time when the textarea is using a slot and twice for when it's using a label prop. This isn't related to the ticket but it would be good to create a spike ticket for further investigation.

Reply:
How are you testing this? I tested this with VoiceOver on macOS (Safari and Chrome) and iOS 16+17 and the screen reader announces the slotted label once.

Reply:
Ah I can reproduce this on Firefox. However, I can reproduce this outside of Ionic too so I think this may be a platform bug (either with Firefox or VoiceOver). The following reads "Hello World, edit text, Hello World, group":

```html
<label>
  Hello World
  <textarea></textarea>
</label>
```

However, the following reads "Hello World, edit text, main":

```html
<label for="text">Hello World</label>
<textarea id="text"></textarea>
```

Reply:
Filed a bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1838502

Reply:
Interesting indeed. I agree that it's an external bug. Then I vote to create a spike to figure out where and submit a bug ticket to them if there isn't one.

Reply:
I can verify that this is also happening with `ion-radio`. Works great for Chrome but not Firefox.